devicetypes/smartthings/zwave-dual-switch.src/zwave-dual-switch.groovy

```diff
@@ -164,13 +181,13 @@ def sendCommand(endpointDevice, commands) {
 	if (commands instanceof String) {
 		commands = commands.split(',') as List
 	}
-	result = commands.collect { encap(endpointNumber, it) }
+	result = commands.collect { command(encap(endpointNumber, it)) }
 	sendHubCommand(result, 100)
 }
 
 def encap(endpointNumber, cmd) {
 	if (cmd instanceof physicalgraph.zwave.Command) {
-		zwave.multiChannelV3.multiChannelCmdEncap(destinationEndPoint: endpointNumber).encapsulate(cmd).format()
+		zwave.multiChannelV3.multiChannelCmdEncap(destinationEndPoint: endpointNumber).encapsulate(cmd)
```

Comment:
at this point you might be better served just wrapping the result of this function in a `command()` call since it's used every place `encap()` is.

Reply:
Yes, that would be a bit cleaner. I'l fix that up.